src/main/kotlin/org/malachite/estella/services/RecruitmentStageService.kt

```diff
@@ -0,0 +1,20 @@
+package org.malachite.estella.services
+
+import org.malachite.estella.commons.models.offers.RecruitmentStage
+import org.malachite.estella.commons.models.offers.StageType
+import org.malachite.estella.process.domain.RecruitmentStageRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@Service
+class RecruitmentStageService(
+    @Autowired private val repository: RecruitmentStageRepository
+) {
+
+    fun save(stage: RecruitmentStage) = repository.save(stage)
+
+    fun save(type: StageType) = repository.save(RecruitmentStage(null, type))
+
+    fun delete(stage: RecruitmentStage) = stage.id?.let { repository.deleteById(it) }
+
```

Comment:
Nie wiem czy nie dodałbym od razu findById

Reply:
Jeśli nie używamy, to po co? Nie Ty mówiłeś, że "przedwczesna optymalizacja jest źródłem wszelkiego zła"? XDDD